calour/manipulation.py

```diff
@@ -224,8 +224,8 @@ def join_experiments(exp: Experiment, other, field_name='experiments', prefixes=
     # when both experiments contain the same sample ids)
     if len(exp.sample_metadata.index.intersection(other.sample_metadata.index)) > 0:
         if prefixes is None:
-            raise ValueError('You need provide prefix to add to sample IDs '
-                             'because the two experiments has some same sample IDs.')
+            raise ValueError('You need provide prefixes=() to add to sample IDs '
```

Comment:
confusing. reading this error msg, I would literally put `prefix=()` in the function call. just use \`prefix\`